packages/odyssey-react-mui/src/iconDictionary/index.ts

```diff
@@ -10,167 +10,227 @@
+import { DevicesIcon } from "./Devices";
+import { DirectoryIcon } from "./Directory";
+import { DocumentationIcon } from "./Documentation";
 import { DownloadIcon } from "./Download";
-export * from "./Download";
-
 import { DragHandleIcon } from "./DragHandle";
-export * from "./DragHandle";
-
 import { EditIcon } from "./Edit";
-export * from "./Edit";
-
+import { ExpandLeftIcon } from "./ExpandLeft";
+import { ExpandRightIcon } from "./ExpandRight";
 import { ExternalLinkIcon } from "./ExternalLink";
-export * from "./ExternalLink";
-
-import { EyeOffIcon } from "./EyeOff";
-export * from "./EyeOff";
-
 import { EyeIcon } from "./Eye";
-export * from "./Eye";
-
+import { EyeOffIcon } from "./EyeOff";
 import { FilterIcon } from "./Filter";
-export * from "./Filter";
-
+import { FolderIcon } from "./Folder";
 import { GlobeIcon } from "./Globe";
-export * from "./Globe";
-
+import { GridIcon } from "./Grid";
+import { GroupIcon } from "./Group";
 import { HomeIcon } from "./Home";
-export * from "./Home";
-
 import { InformationCircleFilledIcon } from "./InformationCircleFilled";
-export * from "./InformationCircleFilled";
-
 import { InformationCircleIcon } from "./InformationCircle";
-export * from "./InformationCircle";
-
+import { LinkIcon } from "./Link";
+import { ListIcon } from "./List";
+import { LockIcon } from "./Lock";
 import { NotificationIcon } from "./Notification";
-export * from "./Notification";
-
 import { OverflowVerticalIcon } from "./OverflowVertical";
-export * from "./OverflowVertical";
-
+import { PauseIcon } from "./Pause";
 import { QuestionCircleFilledIcon } from "./QuestionCircleFilled";
-export * from "./QuestionCircleFilled";
-
 import { QuestionCircleIcon } from "./QuestionCircle";
-export * from "./QuestionCircle";
-
+import { RefreshIcon } from "./Refresh";
+import { ResetIcon } from "./Reset";
+import { ResumeIcon } from "./Resume";
 import { SearchIcon } from "./Search";
-export * from "./Search";
-
+import { ServerIcon } from "./Server";
 import { SettingsIcon } from "./Settings";
-export * from "./Settings";
-
 import { SubtractIcon } from "./Subtract";
-export * from "./Subtract";
-
-import { UserGroupIcon } from "./UserGroup";
-export * from "./UserGroup";
-
+import { SyncIcon } from "./Sync";
+import { UnlockIcon } from "./Unlock";
+import { UploadIcon } from "./Upload";
 import { UserIcon } from "./User";
+import { VideoIcon } from "./Video";
+import { WarningFilledIcon } from "./WarningFilled";
+import { WarningIcon } from "./Warning";
+
+export * from "./Add";
+export * from "./AddCircle";
+export * from "./Apps";
+export * from "./ArrowDown";
+export * from "./ArrowLeft";
+export * from "./ArrowLowerLeft";
+export * from "./ArrowLowerRight";
+export * from "./ArrowRight";
+export * from "./ArrowUp";
+export * from "./ArrowUpDown";
+export * from "./ArrowUpperLeft";
+export * from "./ArrowUpperRight";
+export * from "./Bug";
+export * from "./Calendar";
+export * from "./Call";
```

Comment:
If we have these, I don't think we need to individually import the icons.

But for now, that's fine. In the future, we need to remove the `iconsDictionary` object and make this file a separate Odyssey export. I think there's a ticket for it.